lib/MusicBrainz/Server/Controller/Work.pm

```diff
@@ -153,14 +154,21 @@ sub _merge_load_entities
     my ($self, $c, @works) = @_;
     $c->model('Work')->load_meta(@works);
     $c->model('WorkType')->load(@works);
-    if ($c->user_exists) {
-        $c->model('Work')->rating->load_user_ratings($c->user->id, @works);
-    }
     $c->model('Work')->load_writers(@works);
     $c->model('Work')->load_recording_artists(@works);
     $c->model('WorkAttribute')->load_for_works(@works);
     $c->model('Language')->load_for_works(@works);
     $c->model('ISWC')->load_for_works(@works);
+
+    my @works_with_iswcs = grep { $_->all_iswcs > 0 } @works;
+    if (@works_with_iswcs > 1) {
+        my ($comparator, @tail) = @works_with_iswcs;
+        my $get_iswc_set = sub { Set::Scalar->new(map { $_->iswc } shift->all_iswcs) };
+        my $expect = $get_iswc_set->($comparator);
+        $c->stash(
+            iswcs_differ => any { $get_iswc_set->($_) != $expect } @tail
```

Comment:
You should still add `()` around the `any` call, because if you add more keys after `iswcs_differ`, they'll be consumed by `any`:

```
$ use List::AllUtils qw( any );
$ (foo => any { $_ } (0, 0, 0), bar => 'haha');
$VAR1 = 'foo';
$VAR2 = 1;
$ (foo => (any { $_ } (0, 0, 0)), bar => 'haha');
$VAR1 = 'foo';
$VAR2 = '';
$VAR3 = 'bar';
$VAR4 = 'haha';
```

```suggestion
            iswcs_differ => (any { $get_iswc_set->($_) != $expect } @tail),
```

Reply:
Made the change to both this and the ISRC version in a separate commit.